readme.md

```diff
@@ -64,6 +64,7 @@ $ np --help
     --release-draft-only    Only opens a GitHub release draft
     --test-script           Name of npm run script to run tests before publishing (default: test)
     --no-2fa                Don't enable 2FA on new packages (not recommended)
+    --message               Version bump commit message, '%s' will be replaced with version (default: <version>)
```

Comment:
```suggestion
    --message               Version bump commit message. `%s` will be replaced with version. (default: <version>)
```

Reply:
I'm not sure if `(default: <version>)` makes it clear enough what the default is.

Reply:
About adding the periods: I've added none and used `,` here because all the other flag explanations also didn't use a period at the end. Should I add periods to all of those in another pull request? I'd definitely support that after some research how other CLIs are formatting these lists.


> I'm not sure if `(default: <version>)` makes it clear enough what the default is.

Everything else felt too long, maybe `(default: '%s' with npm or 'v%s' with yarn` is good here?

Reply:
> Everything else felt too long, maybe `(default: '%s' with npm or 'v%s' with yarn` is good here?

I've added that now in the latest commit.